### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by gundalow**
##### SUMMARY
This fixes a few minor issues I found when reviewing the `ce_*` modules, though didn't want to bounce the PR though another review cycle

1) Examples: Remove double space between "option:" and "value"
2) Documentation: Fix rendering usses
3) Documentation: Remove `required: true` to match argspec

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
ce_*

##### ANSIBLE VERSION
```
2.4
```

### CODE REVIEW

File: lib/ansible/modules/network/cloudengine/ce_command.py [hunk 1]
```diff
@@ -52,7 +52,6 @@
         by the configured retries, the task fails.  See examples.
     required: false
     default: null
-    aliases: ['waitfor']
```
Comment (gundalow) [inline lib/ansible/modules/network/cloudengine/ce_command.py:55]: `waitfor` was the original option name in the previous `*_command` network modules. In Ansible 2.2 we standardised on `wait_for`. The alias was added in to allow backwards compatibility,

As this is a new module you only need to support the new option of `wait_for`

**Comment by gundalow** [comment]
@QijunPan This PR fixes a few issues I found when reviewing your PR and doing a local docs build.

I thought it would be easier for me to fix these issues post merge, rather than you having to go through another review cycle.

You may find the diff with whitespaces ignored `?w=1` easier to read https://github.com/ansible/ansible/pull/25059/files?w=1

The only things I notice, that haven't addresses as they would require you to update the code and retest are:

1) In `lib/ansible/plugins/action/ce.py`
In other network modules in that directory we have

`pc.private_key_file = provider['ssh_keyfile'] or self._play_context.private_key_file`

Could you please test that you can use a ssh key (and no `password:`) to successfully connect to the device using Ansible playbooks?


2) There appears to be some duplicated code in some modules, an example would be `get_interface_type` I wonder if that (and other cases) should move to `lib/ansible/module_utils/ce.py`?


A single PR that does all the refactoring across all your modules is fine, no need for separate PRs

**Comment by QijunPan** [comment]
Thank you very much for your handling of these issues, it's makes the document more formal. 
for you question：
1.We're not currently using ```ssh_keyfile```  to connect to the device, but we'll try to implement it.
2.Yes, it's necessary to remove the duplicated code, and I'll update it.

**Comment by gundalow** [comment]
My pleasure, thank you for your patience while we got your modules reviewed and merged. 

I'll merge this then and let you know once the docs.ansible.com has been updated.

**Comment by gundalow** [comment]
The docs have been rebuilt. 

http://docs.ansible.com/ansible/list_of_network_modules.html#cloudengine

At your convenience, please review the above, and if there are any isn't with content if formatting they can be addressed in a single PR. 

Kind regards, 
John Barker

**Comment by QijunPan** [comment]
That seems ok, thanks.

**Pull request merged**